messages/env.compute.collaborator.add.md

```diff
@@ -0,0 +1,13 @@
+# summary
+
+Add a Heroku user as a collaborator on compute environments, allowing them to attach add-ons.
+
+# examples
+
+- Add a Heroku user to all compute environments:
```

Comment:
To be consistent with the description:

`Add a Heroku user as a collaborator on this Functions account.`